src/RealityDataClient.ts

```diff
@@ -70,7 +72,7 @@ export interface RealityDataResponse {
  */
 export class RealityDataAccessClient implements RealityDataAccess {
 
-  public readonly baseUrl: string = "https://api.bentley.com/realitydata";
+  public readonly baseUrl: string = "https://api.bentley.com/reality-management/reality-data";
```

Comment:
did the scopes change?

Reply:
scopes are the same `realitydata:read`  `realitydata:modify`